styles/default.css

```diff
@@ -4,6 +4,45 @@ body {
   font-family: "Titillium Web", sans-serif;
   color: #17324d;
 }
+ 
+.showcasePadding .MuiContainer-root {
+  padding-left: 32px;
+  padding-right: 32px;
+}
+
+#ritiroPagination {
+  .MuiTablePagination-toolbar {
+    padding: 0;
+
+    .MuiTablePagination-actions,
+    .MuiTablePagination-selectLabel,
+    .MuiTablePagination-displayedRows {
+      display: none;
+    }
+
+    .MuiTablePagination-select {
+      padding-left: 0;
+    }
+  }
+}
+
+#iconLeggi {
+  width: 5rem;
+}
+
+.MuiPopperUnstyled-root .MuiAutocomplete-paper {
+  box-shadow: 5px 5px 20px 0px #e1e0e0;
+}
+
+#ritiroPagination_page_mobile ul {
+  justify-content: flex-end;
+}
+
+.MuiOutlinedInput-root:has(#textFilter) {
+  padding-right: 0;
+}
+
```

Comment:
remove extra line 44 and 45

Reply:
Is needed in the file "pages\punti-di-ritiro\index.tsx". Is used to remove  padding from the magnifying glass icon in the search bar